cmd/infracost/output.go

```diff
@@ -143,6 +143,33 @@ func outputCmd(ctx *config.RunContext) *cobra.Command {
 			if cmd.Flags().Changed("fields") && !contains(validFieldsFormats, format) {
 				ui.PrintWarning(cmd.ErrOrStderr(), "fields is only supported for table and html output formats")
 			}
+
+			if ctx.Config.EnableDashboard {
+				if ctx.Config.IsSelfHosted() {
+					ui.PrintWarning(cmd.ErrOrStderr(), "The dashboard is part of Infracost's hosted services. Contact hello@infracost.io for help.")
+				}
```

Comment:
do we want to return here - or is it enough to just flag a warning?

Reply:
Warning is enough, the client won't make a request, but we still want the results. We do the same in the breakdown/diff commands.